lib/src/client.dart

```diff
@@ -537,6 +539,37 @@ class Client {
     PaginationParams paginationParams = const PaginationParams(limit: 10),
     int messageLimit,
     bool onlyOffline = false,
+  }) {
+    final hash = base64.encode(utf8.encode(
+        '$filter${sort?.map((s) => s.toJson())}$options${paginationParams?.toJson()}$messageLimit$onlyOffline'));
+
+    if (_queryChannelsStreams.containsKey(hash)) {
+      return _queryChannelsStreams[hash];
+    }
+
+    final s = _doQueryChannels(
```

Comment:
Consider using a more descriptive variable name for `s`. At a glance, it's difficult know what this value represents or it's intended function :)